portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -66,6 +66,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     boolean readableComment = false;
     boolean readableUsername = false;
 
+    int limit = 10000;
```

Comment:
We usually prefer named constants over the literals (create the constant with meaningful name and store it somewhere in the class). Like TOTAL_COMMENTS_SIZE_LIMIT (I hope linter will tell you how you should name constants, but I guess it should be in caps lock)

Reply:
Oh, I will fix. But linter didn't give an error.